client/src/components/PageLayout.tsx

```diff
@@ -75,13 +75,21 @@ export function AdminPageLayout({
   loading,
   children,
   courses,
-}: PropsWithChildren<{ session: Session; title?: string; courseName?: string; loading: boolean; courses: Course[] }>) {
+  styles,
+}: PropsWithChildren<{
+  session: Session;
+  title?: string;
+  courseName?: string;
+  loading: boolean;
+  courses: Course[];
+  styles?: Partial<{ margin: string | number; padding: string | number }>;
```

Comment:
I think we can use just
```suggestion
  styles?: React.CSSProperties;
```

Reply:
fixed